src/components/ModuleFields/Configurator/File.vue

```diff
@@ -45,6 +45,138 @@
         {{ $t('kind.file.view.showName') }}
       </b-form-checkbox>
     </b-form-group>
+
+    <hr>
+
+    <h5 class="mb-2">
+      {{ $t('kind.file.view.style') }}
+    </h5>
+
+    <b-row
+      cols="12"
+      class="mb-2 mt-2"
+    >
+      <b-col
+        cols="6"
```

Comment:
This will look broken on smaller screens, try to see how it is responsive (look at number configurator for example)